UserCard.jsx

```diff
@@ -0,0 +1,39 @@
+function UserCard({ userId }){
+  const [user, setUsers] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true)
+  
+  React.useEffect(()=>{
+    getUser(userId)  
+      .then(user =>{
+        setUsers(user);
+        setIsLoading(false);
+      });
+  },[]);
+  
+  
+  return (
+    <div>
+      <ul>
+      {isLoading ? ("...Loading"): 
+        <li key={user.id}>
+        <p>Email: {user.email}</p>
+        <p> {user.first_name}</p>
+        <p>{user.last_name}</p>
+        <img src={user.avatar}/>
+        </li>}
+      </ul>  
+    </div>
+    );
+}
+
+
+async function getUser(userId){
+  try{
```

Comment:
👍 to the `try` / `catch` block here as well. You could also add a `catch` callback in your component to display an error to the user if something goes wrong. 

```jsx

getUser(userId)
  .then(user => setUser(user))
  .catch(error => {
    setError(err)
    setIsLoading(false)
  })
```

Reply:
I did not know that, that looks so more readable also, thank you